modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -805,7 +806,62 @@ def decision_reviews_nod_response_schemas
     }
   end
 
-  def nod_response_schemas = decision_reviews_nod_response_schemas
+  def nod_response_schemas
+    decision_reviews_nod_response_schemas.merge(
+      {
+        'nodCreateResponse': {
+          'description': 'Successful response of a 10182 form submission',
+          'type': 'object',
+          'properties': {
+            'data': {
+              'properties': {
+                'id': {
+                  'type': 'string',
+                  'description': 'Unique ID of created NOD',
+                  'example': '97751cb6-d06d-4179-87f6-75e3fc9d875c'
+                },
+                'type': {
+                  'type': 'string',
+                  'description': 'Name of record class',
+                  'example': 'noticeOfDisagreement'
+                },
+                'attributes': {
+                  'type': 'object',
+                  'properties': {
+                    'status': {
+                      'type': 'string',
+                      'description': 'Status of NOD',
+                      'example': AppealsApi::NodStatus::STATUSES.first,
+                      'enum': AppealsApi::NodStatus::STATUSES
+                    },
+                    'createdAt': {
+                      'type': 'string',
+                      'description': 'Created timestamp of the NOD',
+                      'example': '2020-12-16T19:52:23.909Z'
+                    },
+                    'updatedAt': {
+                      'type': 'string',
+                      'description': 'Updated timestamp of the NOD',
+                      'example': '2020-12-16T19:52:23.909Z'
+                    }
+                  }
+                },
+                'formData': {
+                  '$ref': '#/components/schemas/nodCreate'
+                }
+              }
+            },
+            'included': {
+              'type': 'array',
+              'items': {
+                '$ref': '#/components/schemas/appealableIssue'
```

Comment:
This was another instance (like in supplemental claims) where we'd forgotten to update `contestableIssue` to `appealableIssue`